token-metadata/program/src/processor/metadata/transfer.rs

```diff
@@ -110,8 +110,12 @@ fn transfer_v1(program_id: &Pubkey, ctx: Context<Transfer>, args: TransferArgs)
     if let Some(master_edition) = ctx.accounts.edition_info {
         assert_owned_by(master_edition, program_id)?;
     }
+
+    // if a blank rule set comes in we don't need to check this, catch all
     if let Some(authorization_rules) = ctx.accounts.authorization_rules_info {
-        assert_owned_by(authorization_rules, &mpl_token_auth_rules::ID)?;
+        if !authorization_rules.data_is_empty() {
```

Comment:
When can this happen?

Reply:
If you want to see an example, they're in the test cases, create a pNFT with asset data rule_set: Option<None>